workflows/shallowmapping.nf

```diff
@@ -106,7 +106,8 @@ workflow SHALLOWMAPPING {
     ch_versions = ch_versions.mix(FASTP.out.versions.first())
 
     /* Download the reference databases */
-    DOWNLOAD_REFERENCES(params.biome, params.run_bwa)
+    def host_name = params.biome.split('-')[0]
+    DOWNLOAD_REFERENCES(params.biome, params.run_bwa, host_name)
```

Comment:
seems you do not need `host_name` here as well

Reply:
Agree.